asciiArt.py

```diff
@@ -49,7 +49,8 @@ def editPhoto(photoPath, style):
     """
     img = Image.open(photoPath)
     imgWidth, imgHeight = img.size 
-    img = img.resize((round(imgWidth/3), round(imgHeight/3)))
+    aspect_ratio = imgHeight/imgWidth
```

Comment:
Thanks for this!
Could you change this to camelcase before I merge?

Reply:
Okay!